dspace-rest/src/main/java/org/dspace/rest/common/FilteredCollection.java

```diff
@@ -111,15 +111,15 @@ private void setup(org.dspace.content.Collection collection, ServletContext serv
         
         this.setNumberItemsProcessed(0);
         if (itemFilters.size() > 0) {
-        	Iterator<org.dspace.content.Item> childItems = itemService.findByCollection(context, collection, limit, offset);
+        	Iterator<org.dspace.content.Item> childItems = itemService.findAllByCollection(context, collection, limit, offset);
```

Comment:
Just wanted to verify that this isn't a potential security issue, and that something is authorizing users to see all the items (including withdrawn ones) in the Collection.  Obviously, a change like this needs to be checked for security...we don't want to accidentally return withdrawn items in REST API requests, if you are not authorized to see them.

Reply:
@tdonohue , I ran the following query (1)unauthenticated and then (2)authenticated as admin in a collection with 1 withdrawn item.

/rest/filtered-collections/<coll-uuid>?limit=500&filters=is_withdrawn&expand=items

The withdrawn item count was 0 when unauthenticated, and no withdrawn items were returned.

When authenticated, a count of 1 was returned and the item was embedded in the result.